lib/matplotlib/backend_bases.py

```diff
@@ -2977,6 +2977,66 @@ def _zoom_pan_handler(self, event):
             elif event.name == "button_release_event":
                 self.release_zoom(event)
 
+    def _start_event_axes_pan(self, event):
+        # call "ax.start_pan(..)" on all relevant axes of an event
+
+        def _ax_filter(ax):
+            return ax.in_axes(event) and ax.get_navigate() and ax.can_zoom()
```

Comment:
```suggestion
            return ax.in_axes(event) and ax.get_navigate() and ax.can_pan()
```
pan?

If you are also using it in zoom below, perhaps you want to make this function take a `zoom` or `pan` argument as well to use it for both?